docs/src/main/sphinx/connector/hive.rst

```diff
@@ -966,6 +966,15 @@ Procedures
 * ``system.flush_metadata_cache()``
 
   Flush Hive metadata caches.
+  It may also be called with set of named parameters pointing specific partition.
```

Comment:
Make it clear the procedure can be called with no parameters.

Make it clear the procedure can be called with schema/table name, but without partition-related parameters.

Reply:
👍 just defined both use case in separate bullet points